### ISSUE TITLE
dev-haskell/reducers-3.12.1-r1 is incompatible with dev-haskell/fingertree-0.1.2.1

### ISSUE THREAD

**Issue by crocket** [gentoo-haskell/gentoo-haskell#653]
When I try to install `dev-haskell/reducers-3.12.1-r1`, I see the following error message.

```
src/Data/Semigroup/Instances.hs:11:10: error:
    Duplicate instance declarations:
      instance Measured v a => Semigroup (FingerTree v a)
        -- Defined at src/Data/Semigroup/Instances.hs:11:10
      instance [safe] Measured v a => Semigroup (FingerTree v a)
        -- Defined in ‘Data.FingerTree’
```

`src/Data/Semigroup/Instances.hs` is from `reducers`. `Data.FingerTree` is from `fingertree`.

**Comment by crocket** [comment]
The fix seems to be to upgrade reducers to 3.12.2

**Comment by trofi** [comment]
Pushed reducers bump as: https://github.com/gentoo-haskell/gentoo-haskell/commit/8ebb09e5a9ebe47023d65f3a0c8dad24558fb7f1

Thanks for the report!
